packages/sources/react/src/components/VtmnTextInput/VtmnTextInput.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import clsx from 'clsx';
+
+import '@vtmn/css-text-input';
+
+export interface VtmnTextInputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {
+  /**
+   * Label text linked to the input
+   * @type {string}
+   */
+  label: string;
+
+  /**
+   * Helper text to help the user
+   * @type {string}
```

Comment:
It could be interesting to add `@defaultValue` in TSDoc for optional props and replace `@default` in TSDoc of `VtmnButton` by `@defaultValue` on the same occasion no?

See: https://tsdoc.org/pages/tags/defaultvalue/

```ts
   * @defaultValue false
```

Reply:
On the helper ? I think that's not the case on svelte component 🤔

Reply:
On each prop, yes but we can improve the doc like this no?

Reply:
Indeed but actually i don't really know how to put as default 🤔

Reply:
I made a suggestion ;) 
I look forward to your feedback :)

Reply:
I made a suggestion ;)
I look forward to your feedback :)